src/main/java/org/manifold/compiler/front/ExpressionGraph.java

```diff
@@ -94,16 +95,7 @@
     return edgeList;
   }
 
-  private Scope scope;
-
-  public ExpressionGraph(Scope scope) {
-    this.scope = scope;
-  }
-
-  public void buildFrom(List<Expression> expressions) {
-    for (Expression e : expressions) {
-      e.accept(this);
-    }
+  public ExpressionGraph() {
   }
 
   public void writeDOTFile(File file) throws IOException {
```

Comment:
As a high level point (for a separate pull request) I think we should fact our the DOT file generation into another module